### PR TITLE
feat: adding dockerfile for the frontend

### PR DESCRIPTION
**Pull request by heyMP**
- fixes: rollup build
- feat: adds Dockerfile to build the application using node.js and hosting the application using nginx
- chore: add README

### CODE REVIEW

**Review by lholmquist — APPROVED** [review]
Looks Good.

The only thing that would have been nice, is to use our RH support node.js image,  but it doesn't have yarn installed(corepack enable is broken also in the image).  maybe once we have everything working we can come back to it

File: Dockerfile.frontend [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine as build
```
Comment (mhdawson) [inline Dockerfile.frontend:1]: Is this is part of what we show to customers as an example? If so just wondering if it should use a RH container like ubi

Comment (heyMP) [reply Dockerfile.frontend:1]: What's an RH container alternative that we could try?  Also, is there one for nginx that we could try?

Comment (heyMP) [reply Dockerfile.frontend:1]: Oh sorry, misunderstood that.  Your above comment sounds like a plan 👍

**Pull request merged**